cmd/gitcloner/cmd/cmd.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type GitCloner interface {
+	CloneRepo(opts *Options) error
+}
+
+type Options struct {
+	Repo              string
+	Path              string
+	Branch            string
+	Revison           string
```

Comment:
typo:
```suggestion
	Revision           string
```

Reply:
done in https://github.com/rancher/gitjob/pull/302/commits/ba633e4fceb4550843dd3d4ddc01927b520ac33c